### PR TITLE
add proxx announcement blog

### LINKED ISSUE #845: Blog: 🧀 update
**Steps**
1. Create a draft of your post in Google Docs and share with @robdodson and @meggin. If you'd prefer to skip straight to a PR, move on to step 2.
2. Create a PR for your post. Instructions for creating a PR: https://github.com/GoogleChrome/web.dev/wiki/How-to-write-a-post

### PR DESCRIPTION
**Pull request by kosamari**
Fixes #845  .

Changes proposed in this pull request this adds PROXX announcement blog post

### CODE REVIEW

**Comment by kosamari** [comment]
@surma @jakearchibald @robdodson PTAL

File: src/site/content/en/blog/proxx-announce/index.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+---
+title: Introducing PROXX
+subhead: a game of proximity, inspired by minesweeper
+authors:
+  - kosamari
+date: 2019-05-09
+hero: hero.jpg
+alt: a logo image of PROXX.
+description: |
+  This post is a test to demonstrate all of the components that can go into
```
Comment (robdodson) [inline src/site/content/en/blog/proxx-announce/index.md]: Need a meta description.

File: src/site/content/en/blog/proxx-announce/index.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  - Less than 5 seconds TTI ([time to interactive](https://web.dev/interactive)) on slow 3G
+  - Consistent 60fps animation
+
+## Web Workers
+
+The game consists of 4 main entities, the core game logic, the UI service, the state service, and the animation graphics. Since we knew from the get-go we would have to run heavily animated graphics on the main thread, we moved the game logic and state service to a web worker in order to keep the main thread as free as possible. 
+
+## Build time pre-render
+
+Our UI is built with Preact, as it allows us to hit our aggressive target for an initial payload that is less than 25kb. In order to give a good initial loading experience, we decided to pre-render our 1st view. We prerender at build time using Puppeteer to access the top page and let preact populate the DOM. The resulting DOM is then serialized to HTML and saved as index.html
```
Comment (robdodson) [inline src/site/content/en/blog/proxx-announce/index.md]: Link to preact

Comment (robdodson) [reply src/site/content/en/blog/proxx-announce/index.md]: Link to puppeteer

File: src/site/content/en/blog/proxx-announce/index.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+## Canvas for animation, (invisible) DOM For accessibility
+
+We render the game graphics in a canvas using WebGL. One canvas is responsible for the background animation and another one canvas for the game grid on top. We also have an HTML table with buttons for accessibility reasons, that is on top of both of these canvases, but is made invisible (opacity: 0). Even though what you see is a canvas rendering of the game state, the player is interacting with the invisible DOM table, giving us the ability to attach event listeners and rely on the browser’s focus management.
```
Comment (robdodson) [inline src/site/content/en/blog/proxx-announce/index.md]: Maybe link to WebGL on MDN

File: src/site/content/en/blog/proxx-announce/index.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+By keeping the DOM element in the canvas, we are able to tap into browsers native accessibility features. For example: by setting `role=“grid”` on our game table, screen readers can announce the row and column of the focused cell without us having to implement that.
+
+## Rollup for bundling and code splitting
+
+Our total size for the app comes down to 100KB gzipped. Out of that, 20KB is for the initial payload (index.html). We use Rollup.js for this project. We have shared dependencies between the main thread and our web worker, and Rollup can put these shared dependencies in a separate chunk that only needs to be loaded once. Other bundlers like web pack duplicate the shared dependencies which results in double-loading.
```
Comment (robdodson) [inline src/site/content/en/blog/proxx-announce/index.md]: Link to rollup

Comment (robdodson) [reply src/site/content/en/blog/proxx-announce/index.md]: Make webpack one word

File: src/site/content/en/blog/proxx-announce/index.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+## Supporting feature phones 
+
+Smart feature phones such as KaiOS phones are rapidly gaining popularity. These are very resource constrained devices, but our approach of using web workers whenever we can allowed us to make the experience highly responsive on these phones as well. Since feature phones come with different input interface (d-pad and number keys, no touchscreen), we also implemented key-based interface.
```
Comment (robdodson) [inline src/site/content/en/blog/proxx-announce/index.md]: Link for KaiOS maybe?

**Comment by robdodson** [comment]
If you rebase against master I pushed a sneaky change that will fix the font size in nested lists. I'm landing it in devsite as well but that won't go out until maybe end of week.

**Comment by surma** [comment]
I added visuals. 

@robdodson PTAL and merge if ya please :D

File: src/site/content/en/blog/proxx-announce/index.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+tags:
+  - post
+  - PROXX
+---
+
+The team that brought you [squoosh.app](https://squoosh.app) is back! This time, we built a web-based game called PROXX ([proxx.app](https://proxx.app)). PROXX is a game of proximity inspired by the legendary game Minesweeper. The game is situated in the space and your job is to find out where the black holes are. It works on all kinds of devices — from desktop all the way to feature phones. Users can play the game using a mouse, keyboard, d-pad even with a screen reader.
+
+<figure class="w-figure w-figure--fullbleed">
+<video controls loop muted poster="https://storage.googleapis.com/webfundamentals-assets/proxx-announce-blogpost/poster.jpg">
+  <source src="https://storage.googleapis.com/webfundamentals-assets/proxx-announce-blogpost/kaios_vp8.webm" type="video/webm; codecs=vp8">
```
Comment (robdodson) [inline src/site/content/en/blog/proxx-announce/index.md]: I sent Surma a link to a bucket for web.dev. If possible I'd like to see if we can use that.

**Pull request merged**